xarray/tests/test_computation.py

```diff
@@ -1313,7 +1313,8 @@ def test_vectorize_dask_dtype_meta():
     data_array = xr.DataArray([[0, 1, 2], [1, 2, 3]], dims=("x", "y"))
```

Comment:
this comment makes me think that this test is now useless for new versions of dask? Shall we just skip for newer versions?

I agree with letting dask surface the error and have the user handle it.